lib/changeNIDMtoSPM.m

```diff
@@ -66,6 +66,24 @@
         contrastWeightMatrix = relevantToExcursion(contrastWeightMatrix, exID, exLabels);
     end
     
+    %Get the StatType and statisticMaps
```

Comment:
These changes do not seem to be related to the SPM batch integration. Can you shortly comment on why they appear here? E.g. do they address another issue, is it a bug fix, other?